src/components/generic/OptionsList/OptionsList.scss

```diff
@@ -0,0 +1,44 @@
+.#{$WEBEX_COMPONENTS_CLASS_PREFIX}-options-list {
+  border: var(--wxc-select-list-border);
+  border-radius: 12px;
+  box-shadow: var(--wxc-meeting-box-shadow);
+  background-color: var(--wxc-options-list-background);
+  padding: 8px;
+  width: 100%;
+
+  ul {
+    list-style: none;
+    margin-left: 0;
+
+    li {
+      padding: 0.5rem 0.75rem;
+      border-radius: 0.5rem;
+      display: flex;
+      justify-content: space-between;
+      align-items: center;
+      cursor: pointer;
+      color: var(--wxc-text-color);
+
+      span {
+        display: block;
+        font-size: 1rem;
+        white-space: nowrap;
+        text-overflow: ellipsis;
```

Comment:
can the text overflow?

Reply:
yes, it can. it has nowrap and its parent has max-width: 100%